src/components/structures/MatrixChat.js

```diff
@@ -1022,17 +1029,18 @@ module.exports = React.createClass({
                     page_element = (
                         <RoomView
                             ref="roomView"
-                            roomAddress={this.state.currentRoom || this.state.currentRoomAlias}
+                            roomAddress={this.state.currentRoomAlias || this.state.currentRoomId}
```

Comment:
No - RoomView now works, peeking and everything, whether you pass in a room alias or a room ID.